docs/en/loading/Kafka-connector-starrocks.md

```diff
@@ -116,9 +190,8 @@ The following steps take a self-managed Kafka cluster as an example to demonstra
 ### connector.class                     
 
 **Required**: YES<br/>
-**Default value**: com.starrocks.connector.kafka.SinkConnector<br/>
-**Description**: Class used by this Kafka connector's sink.
-
+**Default value**: <br/>
+**Description**: Class used by this Kafka connector's sink: `com.starrocks.connector.kafka.StarRocksSinkConnector`.
```

Comment:
![image](https://github.com/StarRocks/starrocks/assets/57167462/d0e3f304-86ba-4730-9c76-7e11670ee7aa)
There are two colons in one sentence, which is not generally acceptable.
Is `com.starrocks.connector.kafka.StarRocksSinkConnector` the "class" and cannot be changed? If so, change the sentence to:
**Description**: Class used by this Kafka connector's sink. Se the value to `com.starrocks.connector.kafka.StarRocksSinkConnector`.